providers/shared/components/component.ftl

```diff
@@ -74,6 +74,8 @@
 
 [#assign GLOBALDB_COMPONENT_TYPE = "globaldb" ]
 
+[#assign HEALTHCHECK_COMPONENT_TYPE = "healthcheck" ]
```

Comment:
Looks like this might have bled through from another change. Will let it through as the definition is harmless.